### ISSUE TITLE
Adapter not starting with JS-Controller 6.0.1

### ISSUE THREAD

**Issue by LastPerfectTobi** [iobroker-community-adapters/ioBroker.rpi2#191]
After update JS-Controller to version 6.0.1 beta the adapter rpi2 is not starting with following errors:

```
2024-06-11 08:00:31.447 - info: host.raspberrypi instance system.adapter.rpi2.0 started with pid 10328
2024-06-11 08:00:33.526 - error: host.raspberrypi Caught by controller[0]: /opt/iobroker/node_modules/iobroker.rpi2/main.js:19
2024-06-11 08:00:33.527 - error: host.raspberrypi Caught by controller[0]: const adapter = new utils.Adapter({
2024-06-11 08:00:33.527 - error: host.raspberrypi Caught by controller[0]: ^
2024-06-11 08:00:33.527 - error: host.raspberrypi Caught by controller[0]: TypeError: utils.Adapter is not a constructor
2024-06-11 08:00:33.527 - error: host.raspberrypi Caught by controller[0]: at Object. (/opt/iobroker/node_modules/iobroker.rpi2/main.js:19:17)
2024-06-11 08:00:33.527 - error: host.raspberrypi Caught by controller[0]: at Module._compile (node:internal/modules/cjs/loader:1364:14)
2024-06-11 08:00:33.527 - error: host.raspberrypi Caught by controller[0]: at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
2024-06-11 08:00:33.528 - error: host.raspberrypi Caught by controller[0]: at Module.load (node:internal/modules/cjs/loader:1203:32)
2024-06-11 08:00:33.528 - error: host.raspberrypi Caught by controller[0]: at Module._load (node:internal/modules/cjs/loader:1019:12)
2024-06-11 08:00:33.528 - error: host.raspberrypi Caught by controller[0]: at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
2024-06-11 08:00:33.528 - error: host.raspberrypi Caught by controller[0]: at node:internal/main/run_main_module:28:49
2024-06-11 08:00:33.528 - error: host.raspberrypi Caught by controller[0]: Node.js v18.20.3
2024-06-11 08:00:33.529 - error: host.raspberrypi instance system.adapter.rpi2.0 terminated with code 1 (JS_CONTROLLER_STOPPED)
2024-06-11 08:00:33.529 - info: host.raspberrypi Restart adapter system.adapter.rpi2.0 because enabled
2024-06-11 08:00:48.012 - info: host.raspberrypi "system.adapter.rpi2.0" disabled

```

**Comment by mcm1957** [comment]
@LastPerfectTobi 
Welche Adapetrversion ist betrioffen?

**Comment by LastPerfectTobi** [comment]
> @LastPerfectTobi 
> 
> Welche Adapetrversion ist betrioffen?

1.3.2

**Comment by Garfonso** [comment]
Das sollte in der alpha schon nicht mehr passieren. Hoffentlich kriege ich die Donnerstag so weit, dass sie ins latest kann.

**Comment by mcm1957** [comment]
fixed with 2.0.0

**Comment by Cumulus7** [comment]
> fixed with 2.0.0

Hi,
can you tell me what the fix was?
I get the same problem with the wiffi-wz adapter and would like to fix it. (the adapter seems abandoned since 2021)

THX
Markus

**Comment by mcm1957** [comment]
Maybe simply updateing the dependencies, especially adapter-core, will help in your case.

In addition I have created a Issue to ask if this adapter will be maintained in future. If not we can evaluate to move it to iobroker-community-adapters.
